environment.yml

```diff
@@ -1,32 +1,27 @@
 # A conda environment with all useful package for ctapipe developers
-name: pyirf
+name: pyirf-dev
+
 channels:
   - default
-  - cta-observatory
 dependencies:
-  - ctapipe==0.7
+  - python=3.7
   - astropy
-  - conda-forge::nbsphinx
-  - cython
+  - numpy
   - ipython
-  - joblib
   - jupyter
-  - gammapy=0.8
   - matplotlib
-  - nbsphinx
-  - numba
-  - numpy>=1.15.4
-  - numpydoc
-  - pandas
-  - pytables
-  - pytest
-  - pyyaml
   - scipy
   - setuptools
+  # tests
+  - pytest
+  - pytest-cov
+  # docs
+  - numpydoc
   - sphinx
-  - sphinx-automodapi
   - sphinx_rtd_theme
   - pip
   - pip:
-    - rinohtype
-    - ctaplot
+      - rinohtype
+      - nbsphinx
+      - sphinx_automodapi
+      - uproot
```

Comment:
move to uproot4?

Reply:
I talked with Jim Pivarski recently and uproot should become uproot4 soon. I would wait for that.

Reply:
yes they plan to auto shift uproot to uproot4 this year so without a specific version the env might change without noticing